scripts/ci-e2e.sh

```diff
@@ -8,14 +8,29 @@ export CAPM3PATH="${REPO_ROOT}"
 export WORKING_DIR=/opt/metal3-dev-env
 FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
 
+export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
+
+# Starting from CAPI v1.5.0 version cluster-api config folder location has changed
+# to XDG_CONFIG_HOME folder. Following code defines the cluster-api config folder 
+# location according to CAPM3(since CAPM3 minor versions are aligned to CAPI 
+# minors versions) release branch 
+
+if [[ ${CAPM3RELEASEBRANCH} == "main" ]]; then
+    # Default CAPI_CONFIG_FOLDER to $HOME/.config folder if XDG_CONFIG_HOME not set
+    CONFIG_FOLDER="${XDG_CONFIG_HOME:-$HOME/.config}"
+    export CAPI_CONFIG_FOLDER="${CONFIG_FOLDER}/cluster-api" 
+else
+    export CAPI_CONFIG_FOLDER="${HOME}/.cluster-api"
```

Comment:
we are setting the same var twice in two consecutive lines?

Reply:
In the first line we are getting `XDG_CONFIG_HOME` variable as [CONFIG_FOLDER](url), if it is not set we are defaulting to  `$HOME/.config folder`. 
In the second line we are getting `CAPI_CONFIG_FOLDER` not `CONFIG_FOLDER`. 
For more please check https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
and https://github.com/kubernetes-sigs/cluster-api/pull/6913

Reply:
I somehow didnt see the else :D , sorry ignore this comment.